packages/core/src/navigation/navigation-start.element.scss

```diff
@@ -45,4 +48,17 @@
 
 :host([is-group-start][_active]:not([_expanded])) {
   background: #{$cds-alias-object-interaction-background-active};
+  padding-left: 0;
+  border-left: var(--focus-width) solid var(--cds-alias-object-interaction-background-highlight);
+}
+
+:host([is-group-start][_active]:not([_expanded]):hover),
+:host([_has-focus][is-group-start][_active]:not([_expanded])) {
+  background: #{$cds-alias-object-interaction-background-hover} !important;
+  border: 1px solid var(--cds-alias-object-interaction-background-highlight);
```

Comment:
Another `1px`

Reply:
Updated to `var(--cds-alias-object-border-width-100)`